packages/menu/src/SubMenu.tsx

```diff
@@ -410,7 +334,7 @@ const SubMenu: ExtendableBox<
             </ul>
           )}
         </Transition>
-      </li>
+      </>
```

Comment:
The submenu `ul` is no longer a child of the trigger `li`, but rather a sibling. Do we foresee this causing any issues?

Reply:
Reverted these changes, but aligned how `SubMenu` and `MenuItem`  implement this behavior. Moved the consolidation to #1176